gjspack/test/gjspack.test.js

```diff
@@ -279,7 +332,7 @@ foo/halo.blp
 test("transform error", () => {
   const [, stdout, stderr, status] = GLib.spawn_command_line_sync(
     [
-      "/home/sonny/Projects/troll/gjspack/bin/gjspack",
+      "./bin/gjspack",
```

Comment:
Thanks, could you use https://gitlab.gnome.org/GNOME/gjs/-/blob/master/doc/ESModules.md#importmetaurl to resolve an absolute path?

I don't want tests to depend on cwd